### DISCUSSION TITLE
Mzmine3.4.27 : problems with module dataprocessing filter _rowsFilter.RowsfilterModule

### DISCUSSION THREAD

**Discussion by Frstephanie** [mzmine/mzmine#1340] [Q&A]
Hello everyone

I have installed Mzmine3.4.27 on my computer and I have problems in my pipeline for processing raw LCMSMS data in MzXML format with the "Feature list row filter" module. 
I need to do this step to transfer my data into the MetGem software and when I delete one or more features it allows me to generate a new identifier. My problem is that when I click on this module the window opens with more time (5 or 10 minutes) and when I click to run the module nothing happens. 
Do you have the same problem with other users?
What kind of mistakes should I be making in processing my pipeline?
in advance Thank you very much for your answers
stéphanie B

**Comment by AndreaCa96** [comment]
I also have the same problem, have you installed a recent version of Java? I'm worried that the issues might be related.

**Reply by Frstephanie** [reply]
Thank you for your answer, the download is automatic on my computer.
I feel less alone with this problem  
I will talk to the IT team of my institute to find a solution to go back to the old version of java. 
I hope to be able to come back to my data reprocessing very quickly because I am currently back on version 2.52 and I am having trouble now -;)
Stéphanie

**Reply by AndreaCa96** [reply]
Do you have a macOS? Or Windows?

**Reply by Frstephanie** [reply]
Windows 10 professionnel version 22H2 
my computer is Dell Intel R core TM i7 10875H CPU 2.3 GHz  RMA64 Go 
and you ?

**Reply by AndreaCa96** [reply]
On issues [SteffenHeu](https://github.com/SteffenHeu) reply me, you have to delete the file config, now for me it's ok!

You can try to delete the config file before starting MZmine and then try to open the rows filter module. Does it open faster that way?

https://mzmine.github.io/mzmine_documentation/troubleshooting.html#mzmine-does-not-start

**Reply by Frstephanie** [reply]
Thank you very much for saving my Mzmine3 and my data processing.
